### ISSUE TITLE
The order of sorting transactions is not defined.

### ISSUE THREAD

**Issue by DronMDF** [zold-io/papers#19]
TLDR: In my opinion, we need rules for recognizing transactions as invalid. And the remaining transactions just sorted, so it was beautiful. :)

I see several cases that are not described in the WP.
In chapter 8 the order of mergers is described.
```
2. If the transaction is negative (spending money) and its ID is lower
than the maximum ID in the ledger, it gets ignored as a fraudulent
one (“double spending”);
```
This is not blockchain.  This is not problem. There may be situations where transaction 5 enters the network earlier than transaction 4. I do not see this problem if there is enough money in the wallet. This can happen if the client is decentralized. The identifier does not make sense at all... 

If the value has only the overall balance of the wallet - then we generally can sort everything in time. Although this is not accurate. :)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@dronmdf[/z](https://www.0crat.com/u/dronmdf) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/CB08FTCNQ) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@DronMDF so you are saying that we can get rid of transaction ID entirely?

**Comment by DronMDF** [comment]
@yegor256 No, no...

One wallet can be used on different workstations for payments. It can sharing (For automation payment system for example). Should I require that these nodes support only sequential numbering of transactions? This is superfluous in my opinion. Let in a purse there will be some transactions with identical numbers. It's not a problem if you have enough money.

But the sufficiency of means can be checked more rigidly ... Not on the total balance, but transaction by transactions. For this, it is necessary that the nodes arrange the transactions in the identical order. To avoid unnecessary refusals.
